docs/openapi.json

```diff
@@ -455,6 +455,17 @@
                   },
                   "type": "object"
                 },
+                "maximum_interactive_session_inactivity_period": {
+                  "properties": {
+                    "title": {
+                      "type": "string"
+                    },
+                    "value": {
+                      "type": "string"
```

Comment:
Since this field could have a `None` value it should include `"x-nullable": true` flag similarly as for `kubernetes_max_memory_limit` field

Please not that after this change `reana-client-go` swagger specs will have to be regenerated